Sources/TuistCore/Models/Scheme.swift

```diff
@@ -85,16 +85,35 @@ public class ExecutionAction: Equatable {
     }
 }
 
+public class TargetReference: Equatable {
+    public var projectPath: AbsolutePath
```

Comment:
if we can conform `AbsolutePath` to `Equatable` we can avoid manual conformance to `Equatable`, it appears we compare absolute path in many places, and will need it in future

Reply:
I believe `AbsolutePath` already conforms to `Equatable` through its conformance to `Hashable`. I can change `TargetReference` to a struct and avoid the manual conformance to `Equatable`.